Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+| Audit Logs                          | ✓                                             | ✓                                                                   | ✗                            |
+| Authentication of system identities | ✓ Robot Accounts                              | ✓ Robot Accounts                                                    | ✗                            |
+| Authentication of users             | ✓ Local database, LDAP, OIDC, UAA             | ✓ Local database, LDAP, Keystone, JWT                               | ✓ Local database             |
+| Authorization                       | ✓                                             | ✓                                                                   | ✓                            |
+| Automation                          | ✓ Webhooks                                    | ✓ Webhooks, building images                                         | ✗                            |
+| Vulnerability scanning              | ✓ Trivy, Clair                                | ✓ Clair                                                             | ✗                            |
+| Content Trust and Validation        | ✓ Cosign                                      | ✓ Cosign                                                            | ✗                            |
+| Multi-tenancy                       | ✓ (not on the storage level)                  | ✓ (not on the storage level)                                        | ✓ (not on the storage level) |
+| Backup and restore                  | ✓                                             | ✓                                                                   | ✗                            |
+| Monitoring                          | ✓ Prometheus metrics                          | ✓ Prometheus metrics                                                | ✓ Prometheus metrics         |
+| HA mode                             | ✓                                             | ✓                                                                   | ✗                            |
+| Registry replication                | ✓                                             | ✓                                                                   | ✓                            |
+| Proxy cache                         | ✓                                             | ✓ Feature is in the technology preview stage (non production ready) | ✗                            |
+| Quota management                    | ✓ Based on storage consumption or image count | ✓ Based on storage consumption                                      | ✗                            |
+| Garbage collection                  | ✓                                             | ✓                                                                   | ✗                            |
```

Comment:
It might be worth to see if the GC is a stop the world GC (like docker distribution) or an online one that allows operation during GC run.

Reply:
Harbor has a non-blocking GC from v2.1.0 [release](https://github.com/goharbor/harbor/releases/tag/v2.1.0).

Reply:
In the Quay, there is also [zero downtime](https://docs.projectquay.io/deploy_quay_ha.html#_overview) GC.

Reply:
I incorporated it in 02382f14b5ba7204845eb3e690c4d5b78a49770b